### ISSUE TITLE
can't delete a connector

### ISSUE THREAD

**Issue by sagIoTPower** [SoftwareAG/cumulocity-dynamic-mapper#253]
When you try to delete a connector, HTTP 500 is returned:

2024-09-09 11:49:48.488  INFO 12 --- [p-nio-80-exec-1] d.mapping.rest.MappingRestController     : Tenant XYZ - Delete connection instance lyocx9
2024-09-09 11:49:48.556  INFO 12 --- [p-nio-80-exec-1] d.m.c.core.registry.ConnectorRegistry    : Tenant XYZ - No Client is registered for connector ident lyocx9

**Comment by sagIoTPower** [comment]
Currently only enabled connectors are in the connector registry, hence we run into:
```
AConnectorClient client = connectorRegistry.getClientForTenant(tenant, configuration.getIdent());
if (client == null)
  return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Client with ident" + ident + " not found");
```

**Comment by sagIoTPower** [comment]
This is fixed ind 63817a2cc42481ed221a74d5691ce6cc0ccbad17
